### PR TITLE
Sleepy pens now can't be used in combat

### PR DESCRIPTION
**Pull request by ghost**
# Document the changes in your pull request

Now sleepy pens need 0.5 second channelling to inject reagents in targets, meaning that it can't be used in battle. It is still stealthy and your target doesn't get any messages about it.

Closes https://github.com/yogstation13/Yogstation/pull/14359
# Wiki Documentation

lol

# Changelog

:cl:  
rscadd: Sleepy pens need 0.5 second to use
/:cl:

### CODE REVIEW

File: code/modules/paperwork/pen.dm [hunk 1]
```diff
@@ -162,6 +162,9 @@
 	if(..())
 		if(reagents.total_volume)
 			if(M.reagents)
+				to_chat(user, "<span class='warning'>You begin to injecting [src] contents into [M]</span>")
+				if(!do_after(user, 1 SECONDS, target = M))
```
Comment (tattax) [inline code/modules/paperwork/pen.dm]:
```suggestion
				to_chat(user, span_warning("You begin to injecting [src]'s contents into [M]"))
				if(!do_after(user, 1 SECONDS, M))
```

**Comment by ToasterBiome** [comment]
Who's using these in combat?

**Comment by maxion12345** [comment]
> Who's using these in combat?

Its real only use is penning someone to throw them into space or kill them. You can stab a guy during combat to mute and disorient, and eventually get a free kill if you just run away while the chems work.

One second might be too long for 'normal' use like kidnapping. It'll be a bit obvious that "wtf is this guy doing standing near me with a pen out" and people will probably move before then. It doesn't need to be long, just not instant so it's not combat worthy. 

Half second maybe?

**Comment by ghost** [comment]
> > Who's using these in combat?
> 
> Its real only use is penning someone to throw them into space or kill them. You can stab a guy during combat to mute and disorient, and eventually get a free kill if you just run away while the chems work.
> 
> One second might be too long for 'normal' use like kidnapping. It'll be a bit obvious that "wtf is this guy doing standing near me with a pen out" and people will probably move before then. It doesn't need to be long, just not instant so it's not combat worthy.
> 
> Half second maybe?

Okay

**Comment by TheRyeGuyWhoWillNowDie** [comment]
this is better than my idea and pretty much what I was thinking of doing, nice job.

**Comment by ghost** [comment]
> this is better than my idea and pretty much what I was thinking of doing, nice job.

😉

File: code/modules/paperwork/pen.dm [hunk 2]
```diff
@@ -162,6 +162,9 @@
 	if(..())
 		if(reagents.total_volume)
 			if(M.reagents)
+				to_chat(user, span_warning("You begin to injecting [src]'s contents into [M]"))
+				if(!do_after(user, 5 , M))
```
Comment (Bibby0110) [inline code/modules/paperwork/pen.dm]:
```suggestion
				if(!do_after(user, 0.5 SECONDS, M))
```

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**